.github/workflows/ci-notebooks.yml

```diff
@@ -0,0 +1,29 @@
+name: ci-notebooks
+on:
+  pull_request:
+    paths:
+      - modin/**
+      - examples/**
```

Comment:
I think modin/examples/tutorial/tutorial_notebooks/introduction/** is enough for now.

Reply:
Fixed.